src/cli/mod.rs

```diff
@@ -434,24 +457,63 @@ impl FlakeHubPushCli {
             }
         };
 
+        let commit_count = match revision_info.local_revision_count {
+            Some(n) => n as i64,
+            None => {
+                tracing::debug!(
+                    "Getting revision count locally failed, using data from github instead"
+                );
+                github_graphql_data_result.rev_count
+            }
```

Comment:
hmmm. not a blocker but can we just get rid of it completely? what scenarios does from_git_root fail, and does the rest of the push process still work even in that case?

Reply:
Generally github action clones are shallow if I recall? I'm not sure I remember. I'm hoping to get rid of querying this data entirely.